tests/suite_test.go

```diff
@@ -194,6 +197,22 @@ func (suite *IntegrationTestSuite) SetupSuite() {
 	loggr.Info("Portforwarding service Successfully set up.")
 }
 
+func printConditions(ctx context.Context, dspa *v1alpha1.DataSciencePipelinesApplication, namespace string, client client.Client) string {
```

Comment:
think this is related to above, but we shouldn't have functional non-test code in *_test.go